src/pages/index.tsx

```diff
@@ -506,7 +506,19 @@ export default function Example() {
                     </div>
                   </div>
                   <h2 className='mt-2 pb-4 pt-4 text-base font-normal text-gray-700'>
-                    {spec && `${spec.info.description} `}
+                    {spec &&
+                      `${spec.info.description} Node API uses auth tokens. `}
+                    {spec && (
+                      <a
+                        href='https://docs.celestia.org/how-to-guides/quick-start#get-your-auth-token'
+                        target='_blank'
+                        rel='noopener noreferrer'
+                        className='text-blue-500 hover:text-blue-700 hover:underline'
+                      >
+                        Learn more about setting them up
```

Comment:
personal suggestion to more explicitly refer to this as a link (but it's already blue so don't feel obliged)
```suggestion
                        Learn more about setting them up here
```

Reply:
linking a past issue and why it was done this way. happy to use "here" if it's more clear
https://github.com/celestiaorg/docs/issues/1073